odl/util/phantom.py

```diff
@@ -540,6 +540,87 @@ def rect(x):
     return out.ufunc.minimum(1, out=out)
 
 
+def disc_phantom(discr, smooth=True, taper=20.0):
+    """Return a 'disc' phantom consisting in an disc.
+    This phantom is used in [Okt2015]_ for shape-based reconstruction.
```

Comment:
Additional remark: given that the paper never got published (yet), we should remove the citation, also for the submarine.